docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -262,6 +262,15 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=gamma]
 (Optional, double) 
 include::{docdir}/ml/ml-shared.asciidoc[tag=lambda]
 
+`loss_function`::::
+(Optional, string)
+The loss function used during regression. Available options are `mse` (Mean Squared Error),
+`msle` (Mean Squared Logarithmic Error),  `huber` (Pseudo-Huber loss). Defaults to `mse`.
```

Comment:
Ditto re mean squared logarithmic, whereas huber seems to be a person's name and can be capitalized
```suggestion
`msle` (mean squared logarithmic error),  `huber` (Pseudo-Huber loss). Defaults to `mse`.
```